functions.php

```diff
@@ -7,6 +7,11 @@
  * @package _s
  */
 
+if ( ! defined( '_S_VERSION' ) ) {
+	// Replace the version number of the theme on each release.
+	define( '_S_VERSION', '1.0.0' );
```

Comment:
This PR for underscores.me (https://github.com/Automattic/underscores.me/pull/73) deals with searching and replacing this constant with the theme slug.